cmd/libs/go2idl/deepcopy-gen/main.go

```diff
@@ -28,17 +28,19 @@ limitations under the License.
 // Generation is governed by comment tags in the source.  Any package may
 // request DeepCopy generation by including a comment in the file-comments of
 // one file, of the form:
-//   // +k8s:deepcopy-gen=generate
-// or:
-//   // +k8s:deepcopy-gen=register
+//   // +k8s:deepcopy-gen=package
 //
-// Packages which specify `=generate` will have DeepCopy functions generated
-// into them.  Packages which specify `=register` will have DeepCopy functions
-// generated and registered with in `init()` function call to
-// `Scheme.AddGeneratedDeepCopyFuncs()`.
+// Packages can request that the generated DeepCopy functions be registered
+// with an `init()` function call to `Scheme.AddGeneratedDeepCopyFuncs()` by
+// changing the tag to:
+//   // +k8s:deepcopy-gen=package,register
 //
-// Individual types may opt out of DeepCopy generation by specifying a comment
-// of the form:
+// DeepCopy functions can be generated for individual types, rather than the
+// entire package by specifying a comment on the type definion of the form:
+//   // +k8s:deepcopy-gen=true
```

Comment:
no.  Register is a whole-pkg option for now.  I thought about making this more generic, and even adding whole-file mode as well as whole-pkg, but I was getting pretty far off course at that point.  I had no real use for per-type register, so I left it off.